src/services/branch-manager.js

```diff
@@ -17,6 +17,7 @@ const ERROR_MESSAGE_ENVIRONMENT_NOT_FOUND = 'The environment provided doesn\'t e
 const ERROR_MESSAGE_NO_CURRENT_BRANCH = 'You don\'t have any branch to push. Use `forest branch` to create one or use `forest switch` to set your current branch.';
 const ERROR_MESSAGE_WRONG_ENVIRONMENT_TYPE = 'The environment on which you are trying to push your modifications is not a remote environment.';
 const ERROR_MESSAGE_NO_DESTINATION_BRANCH = 'The environment on which you are trying to push your modifications doesn\'t have current branch.';
+const ERROR_SET_DEV_ENVIRONEMENT_AS_ORIGIN = 'Cannot set a development environment as origin';
```

Comment:
```suggestion
const ERROR_SET_DEV_ENVIRONEMENT_AS_ORIGIN = 'Cannot set a development environment as origin.';
```

Reply:
You can easily remove this, it is useless if you don't rephrase the error coming from the sever.